packit_service/worker/jobs.py

```diff
@@ -290,11 +290,9 @@ def is_packit_config_present(self) -> bool:
         Returns:
             Whether the Packit configuration is present in the repo.
         """
-        if isinstance(
-            self.event, AbstractCommentEvent
-        ) and get_packit_commands_from_comment(
-            self.event.comment,
-            packit_comment_command_prefix=self.service_config.comment_command_prefix,
+        if (
+            isinstance(self.event, AbstractCommentEvent)
+            and self._search_for_handlers_for_comment_commands()
```

Comment:
would it make sense to only call `get_handlers_for_comment` here? The name `_search_for_handlers_for_comment_commands` currently doesn't match with what the method does - it searches handlers for both comments and check reruns and we should not need to search handlers for check reruns here

Reply:
nice catch! reruns aren't comment commands :grin: